autorelease/tag.py

```diff
@@ -24,6 +24,10 @@
     "dotnet",
 ]
 
+REPO_DENYLIST = [
```

Comment:
Consider adding a comment that this is only used in the `autorelease/tag` workflows. It will still allow direct, explicit triggering on repos in this list.

Reply:
Yep, done.